src/index.ts

```diff
@@ -96,6 +98,36 @@ const handleEvent =
           ...getECParams(event),
         }
 
+    // Handle Bing click id
+    // Click id can come from two places:
+    // 1- The URL on landing page after clicking on a Microsoft Ad
+    // it must then be stored in a first-party cookie
+    // 2- the previously created first party cookie
+    // in addition, a suffix gets added to the mscklid param sent to Bing
+    // to determine if user saw multiple ads
+
+    // if no click id param or existing cookie => mscklid param = N
+    // if msclkid cookie value and param values differ => mscklid = param one + suffix = 1
+    // if msclkid cookie value and param values are equal = mscklid + suffix = 0
+    let suffix = 'N'
+
+    if (event.client.url.searchParams.get(CLICK_ID_PARAM)) {
+      suffix =
+        event.client.url.searchParams.get(CLICK_ID_PARAM) ===
+        event.client.get(CLICK_ID_PARAM)
+          ? '0'
+          : '1'
+
+      event.client.set(
+        CLICK_ID_PARAM,
+        event.client.url.searchParams.get(CLICK_ID_PARAM),
+        { expiry: 3 * ONE_MONTH }
```

Comment:
one tiny thing - can we just use the value of three months instead of calculating it on each request?

so `const THREE_MONTHS = 7884000000` instead of `const ONE_MONTH = (1000 * 60 * 60 * 24 * 365) / 12`
and then `{ expiry: THREE_MONTHS }` instead of `{ expiry: 3 * ONE_MONTH }`

Reply:
agree
```suggestion
        { expiry: THREE_MONTHS }
```